.azure-pipelines/azure-pipelines-image-template.yml

```diff
@@ -58,9 +58,23 @@ jobs:
         displayName: 'Make configure'
     postSteps:
       - script: |
+          BUILD_REASON=$(Build.Reason)
+          if [[ "$BUILD_REASON" != "PullRequest" && "$GROUP_NAME" == "vs" ]]
```

Comment:
Azure.sonic-buildimage.official.vs
Please use pipeline name in condition. Only this pipeline need the step.